app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt

```diff
@@ -377,6 +402,27 @@ class ImageRegionSelectionInteractionViewTest {
     }
   }
 
+  @Test
```

Comment:
Could you add a test that confirms the submit button is enabled by default?

Reply:
I will add test for that/